pageserver/src/tenant/timeline.rs

```diff
@@ -1496,6 +1503,8 @@ impl Timeline {
                     &timeline_id,
                 ),
 
+                directory_metrics: array::from_fn(|_| AtomicU64::new(0)),
```

Comment:
The split is quite janky right now, but should this be in TimelineMetrics instead?

Reply:
Since we are only exposing one of these, should we consider exposing these via `GET /v1/tenant/:tenant_id/timeline/:timeline_id`?

Reply:
Done